agent/scripts/newrelic.ini.private.template

```diff
@@ -193,3 +193,12 @@
 ;          hooks instrumented.
 ;
 ;newrelic.framework.wordpress.hooks_skip_filename=""
+
+; Setting: newrelic.distributed_tracing.pad_trace_id
+; Type   : bool
+; Scope  : system
+; Default: false
+; Info   : Pads the trace id to 32 characters. This will increase
+;          the amount of data sent to and therefore billed by New Relic.
```

Comment:
```suggestion
; Info   : If set to true, the internal trace_id generated by the agent will have the length of 32 characters. This will increase
;          the amount of data sent to and therefore billed by New Relic.
```

Reply:
Addressed in 0872a62e692b0f700ce788d9eba30a230487616a.